composer.json

```diff
@@ -39,6 +39,7 @@
         }
     },
     "require": {
+        "php": ">=7.1",
```

Comment:
php 7.1 has only been off support for a few months and is still allowed up to laravel 5.8 (last 5.x version so I want to give it a bit more time)